notebooks/xgboost-model/override_model_predict.py

```diff
@@ -0,0 +1,9 @@
+
+import numpy as np
+import xgboost as xgb
+class TransformedPredict:
+    def __init__(self,model):
+        self.model = model
+    def predict(self,arr):
+        dtest = xgb.DMatrix(data=arr)
+        return self.model.predict(dtest)
```

Comment:
this file should not be checkin, its created in the notebook itself using cell magic, for the users to have see why its needed and not have to open another file. it needs to be separate from notebook coz of the multiprocessing inside notebook issue. We should either keep the file and remove the %%wite override_model_predict.py from the cell or just leave the magic and not check in the file